code/modules/gear_presets/survivors/solaris/preset_solaris.dm

```diff
@@ -79,9 +79,25 @@
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/jackboots(new_human), WEAR_FEET)
 	..()
 
+/datum/equipment_preset/survivor/uscm/solaris
+	name = "Survivor - Solaris United States Colonial Marines Corps Recruiter"
+	assignment = "USCMC Recruiter"
```

Comment:
assignment = "USCM Recruiter"

consistency

Reply:
The assignment is USCMC, like the name.

Reply:
> The assignment is USCMC, like the name.

All jobs use USCM, none use USCMC. It should be USCM Recruiter for consistency.

https://gyazo.com/ae223fc6a2f1f1ae8afdacfdf706644e

Reply:
Done